index.html

```diff
@@ -440,10 +440,11 @@ <h4>Computation steps</h4>
               </ol>
               <div>
                 <summary>Example:</summary>
-                <p>Consider a <a class="role-reference" href="#checkbox">check box</a> label that contains a text input field: "Flash the screen [input] times". If the user has entered "5" for the embedded <a class="role-reference" href="#textbox">textbox</a>, the complete label is "Flash the screen 5 times", e.g.: </p>
-                <pre class="example highlight">
-                  <code>&lt;div role="checkbox" aria-checked="false"&gt;Flash the screen &lt;span role="textbox" aria-multiline="false"&gt; 5 &lt;/span&gt; times&lt;/div&gt;</code>
-                </pre>
+                <p>Consider a <a class="role-reference" href="#checkbox">check box</a> label that contains a text input field: "Flash the screen [input] times". If the input field has the value "5", the complete label is "Flash the screen 5 times", e.g.:</p>
+                <pre class="example highlight"><code>&lt;label for="flash"&gt;
```

Comment:
With enclosing label the for attribute is unnecessary

Reply:
It’s not “necessary” but it isn’t invalid and can be necessary for some legacy AT. No need to remove it

Reply:
As mentioned in my prior comment, removing it would also break the label clickability, and therefore reduce the mainstream usability of the control.

Reply:
> As mentioned in my prior comment, removing it would also break the label clickability, and therefore reduce the mainstream usability of the control.

The clickability of the label is also given without the for attribute, if the label encloses the control

Reply:
> The clickability of the label is also given without the for attribute, if the label encloses the control

This didn't used to be the case, and old _defensive_ web dev techniques die hard. Somebody, somewhere is still running a browser from a decade ago.